.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -99,8 +99,14 @@ body:
 
         #### authd-msentraid configuration
         \`\`\`
-        $(cat 2>&1 /var/snap/authd-msentraid/current/broker.conf | sed -E 's/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/<UUID redacted>/g')
+        $(if [ -f /var/snap/authd-msentraid/current/broker.conf ]; then cat 2>&1 /var/snap/authd-msentraid/current/broker.conf | sed -E 's/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/<UUID redacted>/g'; fi)
```

Comment:
This will result in an empty output if the config file doesn't exist. We want the output
```
cat: /var/snap/authd-msentraid/current/broker.conf: No such file or directory
```

(which is why we redirect stderr to stdout)

Reply:
That makes sense, I thought that if the user uses the google provider we would want to omit this section. I will revert this.

Reply:
thanks!